### DISCUSSION TITLE
accesskey

### DISCUSSION THREAD

**Discussion by Alex-Jordan** [openwebwork/webwork2#1919] [Ideas]
Today an instructor asked me if there were hotkeys for the submit buttons. Aside from just hitting Enter to activate Preview, I don't think that there are.

There is the [accesskey atrribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey). Should we use it? Preview, submit, check, and show-me-another could have `p`, `s`, `c`, and `a`. But I see the accessibility concerns mentioned on that reference page.

**Comment by drgrice1** [comment]
In addition to the problems noted on the MDN page, the accesskey attribute would not work for MathQuill.  Pressing Enter for submit is default browser behavior for native inputs.  For MathQuill that is implemented in mqeditor.js.   So we would need to use accesskey for native inputs, and implement the hotkeys for MathQuill separately.
